### ISSUE TITLE
IOS Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/reaver-wps#425]
```
IOS Support, I mean most of the Apple Devices share a Wifi Chipset, so you port 
to 1 or 2 and you have ported to most of them!

IOS based on Linux, if your Jailbroken BASH is a step away (sauriks repo) and 
it seems all the pre-reqs. are already available.

Please PLease PLEase PLEAse PLEASe PLEASE consider porting to IOS!!! (All 
version plz... I only have a 1st gen iPod...)
```

Original issue reported on code.google.com by `PFCKruto...@gmail.com` on 24 Oct 2012 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
but can the iphone chipset be put into monitor mode? im not sure it can ...
```

Original comment by `cn.robe...@me.com` on 30 Oct 2012 at 3:18

**Comment by GoogleCodeExporter** [comment]
```
There was a version of AirCrack for it at one point I believe.
```

Original comment by `PFCKruto...@gmail.com` on 30 Oct 2012 at 5:43

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```

You can't put the wireless card in monitor mode, and iOS is not based on Linux 
at all. 
Just because an OS runs a gnu shell doesn't mean shit. Do you understand bash 
was first released in 1989, years before Torvalds even released Linux v0.1? It 
was made for RichardStallman's fail GNU OS project.

iOS uses xnu as the kernel just like Darwin/OS X. It's HAL framework is IOKit, 
an implementation of NeXTSTEP's DriverKit but in C++. Other parts are based on 
CMU Mach. The userspace code, networking stack, among other things is based on 
FreeBSD. More importantly, every kernel extension used with xnu in iOS is 
closed source, unlike in Darwin/OS X (though most are closed in it, including 
everything wireless-related.) Do you see any mention of Linux on there? 
Everything listed predates Linux. Anyways...

tldr; You aren't going to be putting an iOS device in monitor mode. Yes there 
is a build of aircrack-ng for it, but that app has nothing to do with sniffing 
wireless networks; just cracking the resulting dump. You can compile that for 
about any POSIX compatible OS.

```

Original comment by `naw...@gmail.com` on 18 Mar 2013 at 1:23
